src/Language/Haskell/Liquid/UX/CmdLine.hs

```diff
@@ -811,6 +811,11 @@ resDocs _ (F.Safe  stats) =
     orHeader   = text $ "LIQUID: SAFE (" <> show (Solver.numChck stats) <> " constraints checked)"
   , orMessages = mempty
   }
+resDocs _k (F.Crash [] s)  =
+  OutputResult {
+    orHeader = text "LIQUID: ERROR"
+  , orMessages = [(GHC.noSrcSpan, text s)]
+  }
 resDocs k (F.Crash xs s)  =
   OutputResult {
     orHeader = text "LIQUID: ERROR" <+> text s
```

Comment:
I notice that this case adds the `text s` to the output of the `messages`, so perhaps it would be good to add that to the `orMessages` too (as like the first message)?  That way we don't get inconsistent messages when there are `xs` or not but `s` is the same.

Reply:
Ah, missed this before merging. Feel free to amend, though we might just go for #1980 now.